components/sts/org.wso2.carbon.identity.sts.mgt.ui/src/main/resources/web/identity-sts/configure.jsp

```diff
@@ -131,13 +132,13 @@ try {
 	                         if (cardIssuer.getSupportedTokenTypes()[i].getSupported()) {
 	                       %> 
 	                       <div>
-	                          <input type="hidden" name='<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType() + "hidden"%>' id='<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType() + "hidden"%>' value="true"/>
-	                          <input type='checkbox' name='<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType()%>' id='<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType()%>' checked='checked' onclick="setValue('<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType()%>','<%=cardIssuer.getSupportedTokenTypes()[i].getTokenType()+"hidden"%>')" /> <%=cardIssuer.getSupportedTokenTypes()[i].getTokenType()%> &nbsp;
+	                          <input type="hidden" name='<%=Encode.forHtmlAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType()) + "hidden"%>' id='<%=Encode.forHtmlAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType()) + "hidden"%>' value="true"/>
+	                          <input type='checkbox' name='<%=Encode.forHtmlAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType())%>' id='<%=Encode.forHtmlAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType())%>' checked='checked' onclick="setValue('<%=Encode.forJavaScriptAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType())%>','<%=Encode.forHtmlAttribute(cardIssuer.getSupportedTokenTypes()[i].getTokenType())+"hidden"%>')" /> <%=Encode.forHtmlContent(cardIssuer.getSupportedTokenTypes()[i].getTokenType())%> &nbsp;
```

Comment:
Each parameter passed to the javascript function should be encoded for javascript attribute here